client/src/renderer/redux/actions.js

```diff
@@ -1 +1,12 @@
 /** @format */
+import { ADDCOMMAND } from "./actionTypes";
+
+export const addCommand = (ID, name, keys, behaviour) => ({
```

Comment:
Please use lower case for `ID`.
```suggestion
export const addCommand = (id, name, keys, behaviour) => ({
```